lib/worker.js

```diff
@@ -312,7 +311,12 @@ class AssetComputeWorker {
                 // at this point, we have the rendition a worker created, available at rendition.path
                 // naming rules are rendition0.extension, rendition1.extension, etc.
                 // put postprocessed rendition in a new file inside the post/ directory
-                const newRendition = new Rendition(rendition.originalInstructions, this.directories.postprocessing, rendition.index);
+                const newRendition = new Rendition(
+                    rendition.originalInstructions,
+                    this.directories.postprocessing,
+                    rendition.index,
+                    IS_PIPELINE_RENDITION
```

Comment:
Why a constant? (It should at least be called `NON_PIPELINE_RENDITION`).

Reply:
define `const pipeline = false` above and use it, similar is done in other place.